src/main/java/com/odysseusinc/arachne/portal/service/analysis/impl/AnalysisFilesSavingServiceImpl.java

```diff
@@ -299,20 +299,22 @@ public String updateAnalysisFromMetaFiles(A analysis, List<MultipartFile> entity
         return null;
     }
 
-    private String generateDialectVersions(ZipOutputStream zos, MultipartFile file) throws IOException {
+    private void generateFilesForEachDialectAndAddToZip(ZipOutputStream zos, MultipartFile file) throws IOException {
         String statement = IOUtils.toString(file.getInputStream(), StandardCharsets.UTF_8);
         String renderedSql = SqlRender.renderSql(statement, null, null);
         String baseName = FilenameUtils.getBaseName(file.getOriginalFilename());
         String extension = FilenameUtils.getExtension(file.getOriginalFilename());
-        DBMSType[] dbTypes = new DBMSType[]{DBMSType.POSTGRESQL, DBMSType.ORACLE, DBMSType.MS_SQL_SERVER,
-                DBMSType.REDSHIFT, DBMSType.PDW};
-        for (final DBMSType dialect : dbTypes) {
-            final String sql = SqlTranslate.translateSql(renderedSql, dialect.getOhdsiDB());
-            final String fileName = baseName + "."
-                    + dialect.getLabel().replace(" ", "-")
-                    + "." + extension;
+        List<DBMSType> dbTypes = Arrays.asList(POSTGRESQL, ORACLE, MS_SQL_SERVER, REDSHIFT, PDW);
```

Comment:
Why we should restrict translations only for presented DBMS? I've found the previous behavior with the same, but?

Reply:
have no idea, for some reason, we get only these particular dialects. I think it was discussed with the customer. I would generate SQL for each DBVSType, but need to get approved for this.

Reply:
fixed